main.cpp

```diff
@@ -1,10 +1,18 @@
+#include <inttypes.h>
 #include "mbed.h"
 
 DigitalOut led1(LED1);
 
 // main() runs in its own thread in the OS
 int main() {
+    uint32_t ctr = 0;
+
+    printf("Starting mbed-os-example-blinky\n");
     while (true) {
+        ctr++;
+        if (ctr % 10 == 0) {
```

Comment:
I understand that there is a board without LED, but then blinky becomes more than just a blinky, as it needs to now explain how to setup serial communication, and what it prints.

Not rather to explain if the board does not have app LED, a user needs to add external LED for this example to be functional?

Reply:
Well, then one has to add those instructions on a board per board basis, as there is no "common way of doing it", I think. RTL8195AM should get those instructions yes.

But, the CI need still exists.